### ISSUE TITLE
[FEATURE] Add ARIA attributes to all forms

### ISSUE THREAD

**Issue by Avinash905** [MilanCommunity/Milan#926]
### Description

**Problem:** Currently, many form components of contact us, log in, and signup on the website lack proper accessibility support. 

**Proposed change:** Adding ARIA (Accessible Rich Internet Applications) attributes to our login, signup, and contact forms to improve accessibility.

**Result:** By adding these ARIA attributes to those forms, we can greatly improve the accessibility of the website and ensure that all users, including those with disabilities, can access and interact with the content effectively.

### 🥦 Browser

All

### Checklist ✅

- [X] I checked and didn't find similar issue
- [X] I have read the [Contributing Guidelines](https://github.com/tamalCodes/Milan/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (blank for no)

**Comment by Avinash905** [comment]
Please assign me this issue so that I can work on this issue under **GSSOC'23**.

**Comment by tamalCodes** [comment]
### Remember, Good PR makes you a Good contributor !

We at Milan work hard to maintain the structure, and use conventional Pull request titles and commits. Without a proper template for the PR, not following the guidelines and spam might get the pull request closed, or banned.


- Please make sure that you have read our documentation on [Commiting guidelines](https://github.com/IAmTamal/Milan/blob/main/CONTRIBUTING.md#commits-) and [Pull request guidelines](https://github.com/IAmTamal/Milan/blob/main/CONTRIBUTING.md#pull-requests-).

- Once you have done the changes you can `request` a review. If I suggest changes again, do those and `request` a review again.

- Make sure you DO NOT modify the PR template that we have, use it as it is.

- Last but not least, be patient. It takes a while to review the changes and get it merged.
